subsys/bluetooth/audio/mcs.c

```diff
@@ -817,7 +861,7 @@ void media_proxy_sctrl_track_changed_cb(void)
 void media_proxy_sctrl_track_title_cb(const char *title)
 {
 	BT_DBG("Notifying track title: %s", log_strdup(title));
-	notify(BT_UUID_MCS_TRACK_TITLE, title, strlen(title));
+	notify_string(BT_UUID_MCS_TRACK_TITLE, title);
```

Comment:
What about media player name?

Reply:
I added a comment here: https://github.com/zephyrproject-rtos/zephyr/issues/41189

Reply:
Group titles and segment names are kept in the object, not as characteristics.

Reply:
I'm not see the aforementioned comment in https://github.com/zephyrproject-rtos/zephyr/issues/41189 :)

> Group titles and segment names are kept in the object, not as characteristics.

Not sure how that relates to the media player name - Can you elaborate?

Reply:
1) It is a note in the description, not a comment
2) No relation - they are other strings, and I just wanted to show that they are not relevant for this PR.